geps/gep-1767/index.md

```diff
@@ -656,6 +644,48 @@ Access-Control-Allow-Headers: DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Request
 Access-Control-Expose-Headers: Content-Security-Policy
 ```
 
+###  Disabling credentials
+
+To disable credentials for cross-origin requests, simply don't set the
+`allowCredentials` field at all. Or if you prefer to be explicit, you can
+set it to `false`:
```

Comment:
```suggestion
`allowCredentials` field at all. If you prefer to be explicit, you can
set it to `false`, although this will generally not be necessary:
```